recipes/libbigwig/all/conanfile.py

```diff
@@ -62,13 +60,9 @@ def validate(self):
             if not zlib_ng.options.zlib_compat:
                 raise ConanInvalidConfiguration(f"{self.ref} requires the dependency option zlib-ng:zlib_compat=True")
 
-        if self.options.with_curl:
-            libcurl = self.dependencies["libcurl"]
-            if libcurl.options.with_imap or libcurl.options.with_pop3 or libcurl.options.with_smtp:
-                raise ConanInvalidConfiguration(f"{self.ref} requires libcurl using the follow options: -o libcurl:with_imap=False -o libcurl:with_pop3=False -o libcurl:with_smtp=False")
```

Comment:
Why is this validation no longer required?

Reply:
The check you're quoting was needed to avoid a bunch of weird undefined references to `gsasl_*` when building `with_curl=True` (see [here](https://github.com/conan-io/conan-center-index/pull/12195#issuecomment-1214193211) for more details).

Because we couldn't make sense of this error, we decided to disable CURL support by default (this is in contrast to the defaults used by upstream), and to add the validation step in question (the validation step was useful because libcurl doesn't link to gsasl when those options are turned off).

Few months back a user reported the same issue [here](https://github.com/conan-io/conan-center-index/issues/15370).
The issue was promptly fixed by the almighty @SpaceIm https://github.com/conan-io/conan-center-index/pull/15373.

With the latest recipe for libcurl I am no longer able to reproduce the link error, so I believe this check is no longer needed.

Reply:
Ahhh thank you so much for the links and detailed description!